shell_util/remote_connection.py

```diff
@@ -45,3 +45,10 @@ def __new__(cls, *args, **kwargs):
         obj.ssh_connect_with_retries(server.ip, server.ssh_username,
                                      server.ssh_password, server.ssh_key)
         return obj
+
+    @staticmethod
+    def delete_info_for_server(server, ipaddr=None):
+        ipaddr = ipaddr if ipaddr else server.ip
```

Comment:
This can be simplified as,
   ipaddr = ipaddr or server.ip

Reply:
I have made the changes in latest commit